packages/tooltip/src/Tooltip/utils/tooltipHandlers.types.ts

```diff
@@ -0,0 +1,95 @@
+import { FocusEventHandler, MouseEventHandler, RefObject } from 'react';
+
+import type { TriggerEvent } from '../Tooltip.types';
+
+export interface CreateTooltipEventsBaseArgs {
+  /**
+   *  The `useState` dispatch method to toggle the tooltip state
+   */
+  setState: React.Dispatch<React.SetStateAction<boolean>>;
+
+  /**
+   * Whether the tooltip event handlers should be enabled
+   */
+  isEnabled?: boolean;
+
+  /**
+   * The delay in milliseconds before the tooltip opens on hover.
+   */
+  delay?: number;
```

Comment:
If consumers use `useTooltipTriggerEventHandlers`, are we giving them control of the delay?

Reply:
feels like a q for design since, I believe, we picked the delay for accessibility reasons.